frontend/src/app/services/answers.service.ts

```diff
@@ -29,6 +29,7 @@ export class AnswersService {
 export interface AnswersPackFilter {
 	idNgo?: number;
 	idObserver?: number;
+	phoneObserver?: number;
```

Comment:
isn't `observerPhone` the correct wording in english?

Reply:
Yes indeed it is. Just went after the pattern from the prps above of putting the `what` before the `who`. 
Never mind :D